package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "js-libs",
   "description": "A collection of JavaScript libraries for dapp development.",
+  "version": "v3.0.26",
```

Comment:
No need to have version on the root package.json, see for example [babel](https://github.com/babel/babel/blob/master/package.json)

Reply:
If I pushed my changes in this branch 'luke-81-electron-3' of js-libs repo without a version number in its package.json, then when I tried to test using this branch 'luke-309-electron-3' of Fether on this line of code https://github.com/paritytech/fether/pull/349/files#diff-52ba44750a2113719866b7d98b31af30R42 `"@parity/electron": "https://github.com/paritytech/js-libs.git#luke-81-electron-3",`, then when I tried to build Fether with `yarn; yarn start` it would give me an error that it required a version number in the package.json file

Reply:
It produced the following:
```
error Can't add "@parity/electron": invalid package version undefined.
```

In the details of the commit message I included the full log https://github.com/paritytech/js-libs/pull/90/commits/8b347e047c260e53f3da0955bf87bb2c10de2744